features/compose_message.feature

```diff
@@ -0,0 +1,20 @@
+Feature: As a visitor.
+  I would like to be able to go to the  inbox and fil in and compose a message.
```

Comment:
Sorry guys.
```
As a visitor
in order to talk with a friend
I would like to compose and send a message
```
or something like that. The second line is the *business objective* - the WHY of this action.